docs/using_modin.rst

```diff
@@ -63,6 +63,42 @@ If you would like to request a particular method be implemented, feel free to `o
 issue`_. Before you open an issue please make sure that someone else has not already
 requested that functionality.
 
+Connecting to a database for `read_sql`
+---------------------------------------
+
+To make Pandas read from a SQL database, you have two options:
```

Comment:
We are trying to adhere lowercase for `pandas`.

Reply:
Done in #3723.